DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/_CourseOptionsExpandable.cshtml

```diff
@@ -0,0 +1,55 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+@model CourseOptionsViewModel
+
+
+<details class="nhsuk-details nhsuk-expander">
+  <summary class="nhsuk-details__summary">
+    <span class="nhsuk-details__summary-text">
+      Course options
+    </span>
+  </summary>
+  <div class="nhsuk-details__text">
+    <dl class="nhsuk-summary-list details-list-with-button">
+      <div class="nhsuk-summary-list__row details-list-with-button__row">
+        <dt class="nhsuk-summary-list__key">
+          Active
+        </dt>
+        <dd class="nhsuk-summary-list__value">
+          @(Model.Active ? "Yes" : "No")
+        </dd>

```

Comment:
Might be worth turning these bool to Yes/No into a partial similar to the value with a dash partial